lms/templates/courseware/program_marketing.html

```diff
@@ -1,3 +1,431 @@
 <%page expression_filter="h"/>
+<%inherit file="../main.html"/>
+<%!
+from datetime import datetime
 
-## This page is intentionally left blank. You can add your own program marketing page using comprehensive theming (http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html?highlight=theming).
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangolib.markup import HTML, Text
+%>
+
+<%namespace name='static' file='../static_content.html'/>
+
+<%
+faq = program['faq']
+program_type = program['type']
+title = program['title']
+status = program['status']
+courses = program['courses']
+subtitle = program['subtitle']
+overview = program['overview']
+instructors = program['instructors']
+job_outlook_items = program['job_outlook_items']
+weeks_to_complete = program['weeks_to_complete']
+full_program_price_format = '{0:.0f}' if program['full_program_price'].is_integer() else '{0:.2f}'
```

Comment:
@marjev Just found a small bug here. We are initializing the program['full_program_price'] with int [here](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/programs/utils.py#L509) if there is no seat in any course run then it will raise an exception.